sample/Kmd.Logic.Cvr.Client.Sample/Program.cs

```diff
@@ -64,7 +64,7 @@ private static async Task Run(AppConfiguration configuration)
                 var configs = await cvrClient.GetAllCvrConfigurationsAsync().ConfigureAwait(false);
                 if (configs == null || configs.Count == 0)
                 {
-                    Log.Error("There are no CVR configurations defined for this subscription");
+                    Log.Information("There are no CVR configurations defined for this subscription");
```

Comment:
Why information while you're going to quit the app without successful result?

Reply:
Yes, by program behavior that's right. I changed as I got suggestion to change it to information as we are able to get successful response but user does not have any configs to work with.